docs: clarify guidelines for becoming a committer (#6364)
expressjs/express: Contributing.md

## Becoming a Committer

> All contributors who making significant and valuable contributions should be on-boarded in a timely manner,

Reviewer: ```suggestion
All contributors who have landed significant and valuable contributions should be onboarded in a timely manner,
```
